crates/websocket/src/handlers.rs

```diff
@@ -249,7 +249,7 @@ impl Handler<CheckCaptcha> for ChatServer {
     let check = self
       .captchas
       .iter()
-      .any(|r| r.uuid == msg.uuid && r.answer == msg.answer);
+      .any(|r| r.uuid == msg.uuid && r.answer.to_lowercase() == msg.answer.to_lowercase());
```

Comment:
@dessalines I didnt know that the captcha verification is in the websocket handler. That means it wont work over http api, right?

Reply:
Registering works over http. The chat server is used here, because its a running service that has the collection of rooms and other things, so its also used to store the "active" captchas, and expire them. https://github.com/lemmynet/lemmy/blob/upgrade_deps_4/crates/api_crud/src/user/create.rs#L73